src/main/java/org/octri/clinchit/repository/ClinicianReviewRepository.java

```diff
@@ -0,0 +1,16 @@
+
+package org.octri.clinchit.repository;
+
+import java.util.List;
+
+import org.octri.clinchit.domain.ClinicianReview;
+import org.octri.clinchit.domain.projections.ClinicianReviewSummary;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource(path = "clinician_review", excerptProjection = ClinicianReviewSummary.class)
```

Comment:
The excerptProject attribute tells the repository to pass back the projection instead of the default domain object in lists.